pkg/services/ngalert/notifier/channels/webhook.go

```diff
@@ -56,11 +67,23 @@ func NewWebHookConfig(config *NotificationChannelConfig, decryptFunc GetDecrypte
 	if url == "" {
 		return nil, errors.New("could not find url property in settings")
 	}
+
+	user := config.Settings.Get("username").MustString()
```

Comment:
Is this function called on save when we create or update a webhook contact point? If it is then that's great!